.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-13]
+        os: [ubuntu-latest, macos-13, macos-14]
```

Comment:
suggestion: could you add a bit more tests? Something like this:
* "install_macos", with a version matrix, picking some foundationdb version that can compile towards macOS in both architectures
* "install_linux": with a version matrix, picking 1 version from each release between 6.3 to 7.3? (you can skip 7.0 and 7.2 as they are abandonned releases)

Reply:
Done!